protocol/contracts/lpp/src/state/total.rs

```diff
@@ -102,19 +112,21 @@ where
         loan_principal_payment: Coin<Lpn>,
         loan_interest_rate: Percent,
     ) -> &Self {
-        self.total_interest_due = self.total_interest_due_by_now(&ctime) - loan_interest_payment;
+        self.total_interest_due = self
+            .total_interest_due_by_now(&ctime)
+            .saturating_sub(loan_interest_payment);
```

Comment:
the use of arithmetics that changes the value deserves at least a comment on why that is safe